src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Clients/AuthorizationApiClient.cs

```diff
@@ -15,14 +15,16 @@ namespace Altinn.Common.PEP.Clients
 {
     public class AuthorizationApiClient
     {
+        private const string SubscriptionKeyHeaderName = "Ocp-Apim-Subscription-Key";
         private readonly HttpClient _httpClient;
         private readonly ILogger _logger;
 
-        public AuthorizationApiClient(HttpClient client, IOptions<PlatformSettings> platformSettings, IOptions<PepSettings> pepSettings, ILogger<AuthorizationApiClient> logger)
+        public AuthorizationApiClient(HttpClient client, IOptions<PlatformSettings> platformSettings, ILogger<AuthorizationApiClient> logger)
         {
             _httpClient = client;
             _logger = logger;
-            client.BaseAddress = new Uri($"{platformSettings.Value.GetApiAuthorizationEndpoint}");
+            client.BaseAddress = new Uri($"{platformSettings.Value.ApiAuthorizationEndpoint}");
+            client.DefaultRequestHeaders.Add(SubscriptionKeyHeaderName, platformSettings.Value.SubscriptionKey);
```

Comment:
Could ass an string.IsNullOrWhitespace before adding the subscription key header if we don't want to send an empty header value.

Reply:
Yeah, I'm a little torn. Maybe leave it the way you implemented it now, for now. It is a theoretical case where we want the empty header if there is something wrong with configuration of an app. Where the key is required.